android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MainActivity.java

```diff
@@ -35,25 +39,18 @@
 public class MainActivity extends Activity {
   private static final String TAG  = "MainActivity";
   private static final int REQUEST_ENABLE_BT = 0;
+  private static final int REQUEST_LOCATION = 1;
   private static final String NEARBY_BEACONS_FRAGMENT_TAG = "NearbyBeaconsFragmentTag";
+  private boolean checkingPermissions =false;
```

Comment:
Also please rename to mCheckingPermissions to indicate this is a member variable